Add 1.20.1 support

nwseOOO opened the pull request.

Commit:
Add 1.20.1 support

Commit:
Change package name

I forgor :skull:

Kamilkime reviewed (commented):
Jak działa, to ładnie wygląda, pan tylko doda puste linijeczki końcowe w każdym pliku

Commit:
Fixed gamebreaking bug

just added unnecessary blank lines at the end of the files because kamilkime said so
